pkg/api/server.py

```diff
@@ -15,24 +22,56 @@ def predict():
 
             p = predictor.Predictor().loadModel(predictionType)
 
-            return p.Predict(dates,logs=logs)
-        @self.app.route('/api/login', methods=['POST'])
+            return p.Predict(dates, logs=logs)
 
+        @self.app.route('/api/login', methods=['POST'])
         def login():
-            """Logic for login endpoint"""
+            username = request.form.get('username')
+            password = request.form.get('password')
+
+            resp = make_response()
+
+            if Authentication.authentication.Login(self, username=username, password=password):
```

Comment:
instead of returning a boolean, you can just return the JSON token; it would be a cleaner Approach.